### ISSUE TITLE
Cannot delete the last item in a `list` in a theme

### ISSUE THREAD

**Issue by artofthesmart** [getgrav/grav#3018]
Looks like there's a glitch where the last item in a `list` theme setting cannot be deleted.

Steps to reproduce:

1.  Install Grav + Admin.
2.  Install the Hypertext theme, a bare-bones theme with a `list` item in the 3rd tab.  [YAML file, see line 266](https://github.com/artofthesmart/hypertext/blob/master/blueprints.yaml#L266).
3.  Add a few off-site links.  Save.
4.  Delete all but one link.  Save.
5.  Delete the last link.  Save.

**Expected Behavior**: The list will be saved as empty.
**Actual Behavior**: The page refreshes and the last list item is still in the list.

The last list item cannot be deleted.

**Comment by mahagr** [comment]
Yes, this is known bug as when the field gets empty, it sends no data back, which is considered to be "field was not sent".

This bug has already been fixed in pages in Grav 1.7, which uses Flex Objects and Form plugin, which has the fix.
